plutus-core/plutus-core/stdlib/PlutusCore/StdLib/Data/Bool.hs

```diff
@@ -22,15 +22,15 @@ import PlutusCore.StdLib.Data.Unit
 import Universe
 
 -- | 'Bool' as a PLC type.
-bool :: uni `Includes` Bool => Type TyName uni ()
+bool :: uni `Includes` Bool => Type tyname uni ()
```

Comment:
I think you just introduced a whole lot of git conflicts for my half-ready PR... Why the change?

Reply:
These changes in stdlib are contained in this commit: https://github.com/input-output-hk/plutus/pull/5390/commits/147e2ee78e0ac936e3f1bd12558d587058e5896f  . I did this to generalize a bit the stdlib to help me reuse parts of it in my debruijn tests. The stdlib otherwise is not "debruijn friendly" (relies on renaming a lot)

Reply:
Eek, that's a very valid reason to introduce the changes. OK, let's keep them and I'll deal with the git conflicts and the problem that it's going to be hard to unhardcode `Type` from the new machinery. Thanks for motivating me them!